main.py

```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+from pandas.core.interchange.dataframe_protocol import DataFrame
```

Comment:
This line is unnecessary because we imported all features of pandas with `import pandas as pd`